src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImportsFromStaticMethodRef.java

```diff
@@ -0,0 +1,94 @@
+/*
+UnusedImports
+processJavadoc = (default)true
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
+
+import static java.lang.annotation.ElementType.TYPE_USE; // ok
+import static java.lang.annotation.RetentionPolicy.RUNTIME; // ok
+
+import java.lang.annotation.Retention; // ok
+import java.lang.annotation.Target; // ok
+import java.lang.ref.WeakReference; // ok
+import java.util.ArrayList; // ok
+import java.util.Optional; // ok
+import java.util.Objects; // ok
+import java.util.Arrays; // ok
+import java.util.List; // ok
+import java.util.Map; // ok
+import java.util.Collections; // ok
+import java.util.function.Consumer; // ok
+import java.util.function.Function; // ok
+import java.util.function.Supplier; // ok
+import static java.lang.String.format; // violation
+import static java.util.Arrays.sort; // violation
+import static java.util.List.of; // violation
+import static java.util.Collections.emptyMap; // violation
```

Comment:
Please put part violation message in comment, `// violation 'unused import .*`

We already demand this for all new tests.
Looks like this Check is not migrated yet.

Reply:
You are right, the checker is not migrated yet. So adding the message will result in failure. It can be tracked as  a separate issue.

Reply:
@renjith7189 always allow reviewers to resolve their own comments. Please add violation message to all comments as requested by @romani

Reply:
@nrmancuso Thanks. For the violation message, adding the additional text causes test to fail. The Checker is not migrated yet it seems and I would suggest to keep track of that as a separate issue https://github.com/checkstyle/checkstyle/issues/11214

Reply:
Please share example of failure.
All Checks already support definition of message in trailing comment.
Example of update just to define message https://github.com/checkstyle/checkstyle/commit/9857df8546c56f48b68fc5e930caffcc70eafe01

There should be no problem in stead of `// violation` to have `// violation 'Unused import .*'`
We just need to make sure that message regexp is matching actual message of violation.

Reply:
https://github.com/checkstyle/checkstyle/blob/13481f2c410e4944ecf5ab93ec49948a523a0c82/src/main/resources/com/puppycrawl/tools/checkstyle/checks/imports/messages.properties#L19